mindmeld/rasaconverter.py

```diff
@@ -0,0 +1,545 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Cisco Systems, Inc. and others.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains the Rasacoverter class used to convert Rasa projects
+into Mindmeld projects"""
+
+from keyword import iskeyword
+import re
+import os
+import copy
+import yaml
+
+from converter import Converter
+
+
+class Rasaconverter(Converter):
+
+    def __init__(self, rasa_project_directory, mindmeld_project_directory):
+        if not os.path.exists(os.path.dirname(rasa_project_directory)):
+            self.rasa_project_directory = rasa_project_directory
+            self.mindmeld_project_directory = mindmeld_project_directory
+
+    @staticmethod
+    def __create_intents_directories(mindmeld_project_directory, intents):
+        for intent in intents:
+            Converter.create_directory(mindmeld_project_directory + "/domains/default/" + intent)
```

Comment:
```suggestion
            self.create_directory(mindmeld_project_directory + "/domains/default/" + intent)
```

Reply:
It's giving me an error when I try this. Is that because it's a static method in the parent class?

Reply:
what's the error?

Reply:
It says (Undefined variable 'self'). Because there is now self argument.

Reply:
You can change this method from static to a normal class method with `self` argument